src/main/scala/uk/gov/nationalarchives/fileformat/FFIDExtractor.scala

```diff
@@ -30,8 +30,11 @@ class FFIDExtractor(sqsUtils: SQSUtils, config: Config) {
       val containerSignatureVersion = signatureOutput(1).split(" ").last
       val droidSignatureVersion = signatureOutput(2).split(" ").last
       val consignmentPath = s"""$efsRootLocation/${file.consignmentId}"""
+      val pathWithQuotesReplaced = file.originalPath.replaceAll(""""""", """\\\"""")
+      val filePath = s""""$consignmentPath/$pathWithQuotesReplaced""""
       //Adds the file to a profile and runs it. The output is a .droid profile file.
-      s"""$command -a  "$consignmentPath/${file.originalPath}" -p $consignmentPath/${file.fileId}.droid""".!!
+      val droidCommand = s"""$command -a  $filePath -p $consignmentPath/${file.fileId}.droid"""
```

Comment:
I have no idea if this will work, but if you split the command up into a Seq, does it remove the need for escaping? https://stackoverflow.com/a/11738575/679924

Reply:
No I just tried it. Definitely worth a try. Because the path is wrapped in double quotes to stop it failing on any other special characters we need to escape any double quotes. Without the escape character, there's one too many. If you were running this in the terminal, you would need the backslash `droid "a path with"\quotes" Without it, it wouldn't work.